### ISSUE TITLE
Using apostrophe in comments

### ISSUE THREAD

**Issue by goncalogato** [openmusic-project/openmusic#63]
Hello,

It seems that using apostrophes inside comments makes OM unable to save a patch. Use the following text inside a comment:

Hit 'm'

...and it causes the issue.

**Comment by karimhaddad** [comment]
Hi, 

Are you sure you are using ASCII apostrophes  and not a glyph you copy pasted ?

**Comment by goncalogato** [comment]
Hello,

I'm using regular characters on my keyboard. Nope, not a glyph.

**Comment by goncalogato** [comment]
Yes, in the comment popup it works. But not while editing the actual comment in the patch. I get this message:

<img width="229" alt="image" src="https://github.com/user-attachments/assets/cfc8839f-b13d-468a-bef2-f29955cce548" />

**Comment by karimhaddad** [comment]
When you edit the comment, what character are you using. For me everything seems ok??
